source/utils.js

```diff
@@ -199,6 +199,21 @@ var utils = {
 
         object[last] = value;
     },
+
+    /**
+     * Returns href from <link rel='canonical'>
+     * if presents in the document
+     *
+     * @returns {String?}
```

Comment:
Is this notation valid? I’ve thought `String|null` should be used in such cases.